core-bundle/src/Event/FilterPageTypeEvent.php

```diff
@@ -14,6 +14,9 @@
 
 use Contao\DataContainer;
 
+/**
+ * @deprecated Deprecated since Contao 5.3, to be removed in Contao 6. Use DCA permissions instead.
```

Comment:
The event is still used in the `PageTypeOptionsListener` listener, so we cannot deprecate it yet.

Reply:
Huh? The `PageTypeOptionsListener` dispatches the event, of course it still needs to do that. **Using** it is deprecated though.